about.css

```diff
@@ -0,0 +1,200 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'lato', sans-serif;
+  box-sizing: border-box;
+}
+
+.about .introduction {
+  background-image: url(./images/mobile-images/mobile-page-bg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  height: 80vh;
+  padding-top: 1.25rem;
+}
+
+.about .introduction-heading {
+  text-align: center;
+}
+
+.about .introduction-paragraph {
+  background-color: #fff;
+  border: 1px solid #e8e8e8;
+  text-align: center;
+}
+
+.about-contact {
+  text-align: center;
+  font-size: 12px;
+}
+
+.about-detail {
+  height: 40rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0.625rem;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.about-heading {
+  padding: 0.625rem;
+}
+
+.about-logo {
+  width: 100%;
+  text-align: center;
+}
+
+.about-logo img {
+  width: 12.5rem;
+  height: auto;
+  border: solid 1px #f1f1f1;
+}
+
+.hide-para {
+  visibility: hidden;
+}
+
+.past-summit {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 35rem;
+}
+
+.past-summits {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  overflow: hidden;
+  gap: 10px;
+}
+
+.past-description {
+  width: 100%;
+  margin: auto;
+}
+
+.past-summit-images {
+  width: 90%;
+  margin: auto;
+  background: #ff4500;
+  overflow: hidden;
+  position: relative;
+  text-align: center;
+  color: white;
+}
+
+.past-img {
+  object-fit: cover;
+  opacity: 0.4;
+}
+
+.past-summit-info {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.past-summit-images img {
+  width: 100%;
+  height: auto;
+}
+
+@media only screen and (min-width: 768px) {
+  .about .past-description {
+    border-top: 2px solid #f1f1f1;
+    padding-top: 10px;
+  }
+
+  .about .introduction-paragraph {
+    background-color: #fff;
+    border: 1px solid #e8e8e8;
+    text-align: center;
+    width: 80%;
+    margin-left: 6.625rem;
+  }
+
+  .about-heading .introduction-paragraph {
+    border: none;
+  }
+
+  .about-detail {
+    height: 40rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 0.625rem;
+  }
+
+  .about-heading h6 {
+    text-align: center;
```

Comment:
- [ ] Kindly make sure your text is centered as it is in the design.
`Hint: You may consider grabbing them inside a container and centering it` :smile: 

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/210800060-44607636-b6aa-4177-998f-16277273801e.png)

Original design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/210800246-ac642b5b-1b74-4066-a66f-77d6add3fd11.png)